src/packages/server/index.js

```diff
@@ -26,9 +27,11 @@ class Server {
 
   router: Server$opts.router;
 
+  cors: Server$opts.server.cors;
```

Comment:
I also feel like the double call here doesn't seem great. As an alternative, I could grab Server$CORS directly. That's probably more sensible but I thought I'd ask.